content/en/observability_pipelines/log_volume_control/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Log Volume Control
+kind: document
+disable_toc: false
+---
+
+## Overview
+
+As your infrastructure and applications grow, so does your log volume and the complexity of the data. A large volume of logs can introduce a lot of noise and make it difficult to analyze and troubleshoot logs. Use Observability Pipelines's processors to decide which logs are valuable and which ones are noisy and uninteresting, before sending your logs to their destinations. You can use the following processors in the Observability Pipeline Worker to manage your logs:
+
+- Filter: Add a query to send only a subset of logs based on your conditions.
```

Comment:
Should this be highlighted a little bit more? Maybe a bold font?

Reply:
Bolded